app/modules/sightings/models.py

```diff
@@ -537,13 +543,29 @@ def ia_pipeline(self, id_configs):
         ]
 
         # No annotations to identify or no algorithms, go straight to un-reviewed
-        if len(encounters_with_annotations) == 0 or num_algorithms == 0:
+        if len(encounters_with_annotations) == 0:
             self.stage = SightingStage.un_reviewed
```

Comment:
@colin, please can you clarify if the individualUuid should be used for encounters without annotations or not?
I chose that if there were no annotations, no individual was possible but I may have this wrong

Reply:
We are going to need to support having an individual for encounters with no annotations. Mostly for researcher datasets that include an encounter of a known individual based on something like a physical tag recording or something. It's still important from a data management use case.